server/database.go

```diff
@@ -20,7 +20,7 @@ const (
 	deploymentUpdateStateStmt    = `UPDATE deployments SET state = ? WHERE deployments.id = ?`
 	deploymentFailUnfinishedStmt = `UPDATE deployments SET state = ? WHERE deployments.state = ? OR deployments.state = ?`
 	lastTargetDeploymentStmt     = `SELECT id, user_id, application_name, target_name, commit_sha, branch, comment, state, created_at FROM deployments WHERE deployments.state = ? AND deployments.application_name = ? AND deployments.target_name = ? ORDER BY created_at DESC LIMIT 1`
-	applicationDeploymentsStmt   = `SELECT id, user_id, target_name, commit_sha, branch, comment, state, created_at FROM deployments WHERE deployments.application_name = ? ORDER BY created_at DESC LIMIT ?`
+	applicationDeploymentsStmt   = `SELECT id, user_id, target_name, commit_sha, branch, comment, state, created_at FROM deployments WHERE deployments.application_name = ? AND deployments.target_name LIKE ? ORDER BY created_at DESC LIMIT ?`
```

Comment:
hmm.. you are kind of misusing the `LIKE` statement in order to save another query definition.

If I get this right, you either want a `deployments.target_name = ?` or you don't want the selector to be applied at all, by passing `%`.

Not sure about it yet, but I think we should look for a better solution here.